interface/main/calendar/modules/PostCalendar/pntemplates/default/user/ajax_search.html

```diff
@@ -62,7 +62,7 @@
 [-if $SEARCH_PERFORMED-]
 <div id="calsearch_results">
```

Comment:
Brought this script into main codebase (PR is at https://github.com/openemr/openemr/pull/1129)